source/MaterialXGenMdl/Nodes/ClosureCompoundNodeMdl.cpp

```diff
@@ -0,0 +1,184 @@
+//
+// TM & (c) 2020 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
```

Comment:
For any new files in the changelist, we should mark the copyright date as "2021", following our convention that copyrights state the year in which the file was contributed to the codebase.